### DISCUSSION TITLE
Is there a way to auto update RVX?

### DISCUSSION THREAD

**Discussion by LeVraiRoiDHyrule** [Android-App-Patches/build-apps#139] [Q&A]
Hi,

I love RVX and this built apk repo is wonderful, it saves a lot of time reapplying settings.

But I am wondering if there is a way to have the whole process automatic. Is there a way to have auto update? Is it possible for example to have an F-Droid repo so I can add this to F-Droid and let it manage updates?

Thanks in advance and have a nice day

**Comment by stu2005** [comment]
Use [Obtainium](https://github.com/ImranR98/Obtainium)
